src/Compiler/SyntaxTree/SyntaxTree.fsi

```diff
@@ -1154,6 +1154,18 @@ type SynAttribute =
 
         /// The syntax range of the attribute
         Range: range
+
+        /// The syntax range of the LESS symbol
+        LessRange: range option
+
+        /// Type arguments
+        TypeArgs: SynType list
+
+        /// Ranges of the commas between type arguments
+        CommaRanges: range list
+
+        /// The syntax range of the GREATER symbol
+        GreaterRange: range option
     }
```

Comment:
It seems to duplicate info from `SynType.App`, `SynType.LongIdentApp` and `SynExpr.TypeApp`. Maybe we could extract these field into a separate type, like `SynTypeArgumentList` or something?

Reply:
So the question is whether we extract all 4 as `SynTypeArgumentList` as just the ranges as `SynTypeArgumentTrivia`.